client/blocks/comments/style.scss

```diff
@@ -49,6 +49,7 @@
 
 			white-space: pre-wrap;
 			word-wrap: break-word;
+			word-break: break-word;
```

Comment:
According to [the CSS spec](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) `break-word` isn't a valid property value. Should it be `break-all` instead?